peerfinder/peerfinder.py

```diff
@@ -185,60 +188,131 @@ def main():
         ix_set = [pdb_to_ixp(ix) for _, ix in ix_dedup.items()]
         netfac_set = [pdb_to_fac(ix) for ix in pdb["data"][0]["netfac_set"]]
         peers.append(pdb_to_peer(pdb["data"][0], ix_set, netfac_set))
+    return(peers)
 
-    if args.ix_only:
-        common_ix_list = fetch_common_ixps(peers)
-        if len(common_ix_list) < 1:
-            print("Didnt find any common IX, exiting...")
-            exit(1)
-        header = []
-        header.append("IX")
+def find_common_points(common_ix_list: List, common_fac_list: List, peers: List) -> Dict:
+    """ Return a dict with all common IXs and facilities
+    Arguments:
+        common_ix_list: A list of all common ixs
+        common_fac_list: A list of all common facilities
+        peers: A list of the involved peers
+    Returns:
+        A dict with the merged IXs and facilites
+    """
+    common_points = {}
+    common_ixs = {}
+    common_facs = {}
+    for ix in common_ix_list:
         for peer in peers:
-            header.append(peer.name)
-
-        ix_tab = PrettyTable(header)
-        ix_tab.print_empty = False
-
-        for ix in common_ix_list:
-            row = [ix]
-            for peer in peers:
-                curr_ix = fetch_ix_from_ixps(ix, peer.peering_on)
-                v4 = "v4: " + "\n".join([str(i) for i in curr_ix.subnet4])
-                v6 = "v6: " + "\n".join(
-                    [str(i) for i in curr_ix.subnet6 if str(i) != "0100::"]
```

Comment:
We use the IPv6 prefix 0100:: to denote that this peer does not have a v6 address. So please ensure you capture this logic here.